aldryn_config.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from aldryn_client import forms
+
+
+def split_and_strip(string):
+    return [item.strip() for item in string.split(',') if item]
+
+
+class Form(forms.BaseForm):
+    templates = forms.CharField(
+        'List of additional templates (comma separated)',
```

Comment:
Same again: we should correct these. Perhaps I should make a list of them all, and then we can tackle them en-masse across the addons? @FinalAngel